railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_1.rb.tt

```diff
@@ -68,3 +68,42 @@
 # (https://open-telemetry.github.io/opentelemetry-sqlcommenter/), or using the legacy format.
 # Options are `:legacy` and `:sqlcommenter`.
 # Rails.application.config.active_record.query_log_tags_format = :sqlcommenter
+
+# Specify the serializer used by `MessageEncryptor` by default
+# Options are `:json`, `:hybrid`, and `:marshal`
+#
+# New apps by default will use `:json`
+#
+# Rails.application.config.active_support.default_message_encryptor_serializer = :json
+#
+# However, if you are upgrading an app you will likely want to start with
+# `:hybrid`. Switching to `:json` before all messages have been converted will
+# result in those messages no longer being deserializable.
```

Comment:
Is it more expected that messages will be converted, or that old messages will just expire?  Or perhaps mention both?

```suggestion
# `:hybrid`. If you switch to `:json` before existing messages have expired or 
# been converted, those messages will no longer be readable.
```